src/port1.0/portconfigure.tcl

```diff
@@ -527,9 +527,17 @@ proc portconfigure::get_compiler_fallback {} {
                 return {gcc-4.0}
             }
         } elseif {[vercmp $xcodeversion 3.0] >= 0} {
-            return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-clang-3.4 macports-clang-3.3}
+            if {${configure.build_arch} eq "ppc" || ${configure.build_arch} eq "ppc64"} {
+                return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-gcc-6  macports-gcc-7}
```

Comment:
There's an extra space there. Why is gcc6 better than gcc7?

Reply:
at present, gcc7 has about 4x as many test failures on PPC as gcc6 does. Also, it's more strict and so raises more unhelpful-to-the-end-user build errors.

Reply:
I remember you mentioned gcc7 problems on PowerPC before, but 7.3.0 was released since then. Does it still have those failures?

Newer clang compilers are also more strict than older clang compilers, yet we don't shun them. We deal with the problems, fix the build failures, report the problems to the developers...

Reply:
I will admit to not having run a full test suite on gcc 7.3.0 on PPC as yet. It's a two-day affair of building and testing :>  I can do that in a bit and recheck the status of the current gcc7, as we might as well get this done right once.